app/flows/check_uk_visa_flow.rb

```diff
@@ -282,7 +282,6 @@ def define
     end
 
     outcome :outcome_diplomatic_business
-    outcome :outcome_joining_family_m
```

Comment:
In your commit, please change `2` to reference the latest commit SHA for the template: https://github.com/alphagov/smart-answers/blob/5e7c9222b41c51e2caa0f480f6350be9b82e8926/app/flows/check_uk_visa_flow/outcomes/outcome_joining_family_m.erb

As otherwise when we look back on this commit, `2` is going to 404.

(Tip: when you're on the 'main' link, you can press the `y` key to automatically convert it to the latest SHA link).

Reply:
Good point Chris 👍 Also thanks for highlighting the `y` Git magic, never knew you could do that!!